server/filters/renderChooseBarcodeOptionRadiosFilter.ts

```diff
@@ -9,7 +9,7 @@ export default function renderChooseBarcodeOptionRadiosFilter(values: Array<stri
         html: `        
           <img src='/assets/images/print-barcodes-image.png' alt='' width='94px'/>
           <details class='govuk-details' data-module='govuk-details'>
-            <summary class='govuk-details__summary'><span class='govuk-details__summary-text'>How it works</span></summary>            
+            <summary class='govuk-details__summary'><span class='govuk-details__summary-text'>How it works <span class='govuk-visually-hidden'> (printing as part of your own document)</span></span></summary>            
```

Comment:
When using a screen reader users can receive a summary of clickable components. In this summary it is not possible to differentiate between the two "How it works" links, so more context was required for each link.